rero_ils/modules/documents/serializers/dc.py

```diff
@@ -16,21 +16,25 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""DublinCore Document serialization."""
+"""RERO Document JSON DublinCore serialization."""
 
 from dcxml import simpledc
 from flask import current_app, request
 from invenio_records_rest.serializers.dc import \
-    DublinCoreSerializer as BaseDublinCoreSerializer
+    DublinCoreSerializer as _DublinCoreSerializer
```

Comment:
cosmetic: in my opinion, it is less telling

Reply:
We use "_" syntax now for other resource that we could override. See: https://github.com/rero/rero-ils/blob/staging/rero_ils/modules/serializers.py#L21-L24 (staging branch).
I just try to harmonize imports... and it's an huge job !